### ISSUE TITLE
Not filtering fields with multiple sub-resources

### ISSUE THREAD

**Issue by Dragomir-Ivanov** [rs/rest-layer#148]
Trying to get multiple sub-resources and selecting its fields on each of them, doesn't work as expected.

My concrete example:
```
GET http://192.168.0.103:3001/api/clients/59a962a5eaba9ebd0180ac67?fields=id,name,address,telephones,notes,visits,services{id,createdDT,status,name,zipCode,insuraceCompany,workOrder,ownOrder},appliances
```
, doesn't filter the fields of the `services`. If I remove the `appliances` sub-resource, it filters them.

**Comment by Dragomir-Ivanov** [comment]
PR supplied #154

**Comment by Dragomir-Ivanov** [comment]
Resolved by #154
